chef/cookbooks/ceilometer/recipes/mongodb.rb

```diff
@@ -69,7 +69,7 @@
     members = search(:node,
       "ceilometer_ha_mongodb_replica_set_member:true AND "\
       "ceilometer_config_environment:#{node[:ceilometer][:config][:environment]}"
-      ).sort
+      ).sort { |a, b| a.name <=> b.name }
```

Comment:
the sort is pointless, configure_replicaset does that already (and properly). just remove it.